basicswap/templates/wallet.html

```diff
@@ -1,515 +1,792 @@
-                        <button type="submit" class="flex flex-wrap justify-center py-2 px-4 bg-blue-500 hover:bg-blue-600 font-medium text-sm text-white border border-blue-500 rounded-md shadow-button focus:ring-0 focus:outline-none" name="estfee_{{ w.cid }}" value="Estimate Fee">Estimate Fee</button>
-                        {% endif %} {# / XMR #}
-                      </td>
-                    </tr>
-                  </table>
+                      </div>
+                      {% endif %}
+                      {% if w.cid == '3' %}
+                      {# LTC #}
+                      <div class="w-full md:w-1/2 p-3 flex justify-center items-center">
+                        <div class="h-full">
+                          <div class="flex flex-wrap -m-3">
+                            <div class="w-full p-3">
+                              <div class="mb-2 qrcode-container flex h-60 justify-center items-center">
+                                <div class="qrcode-border flex">
+                                  <div id="qrcode-mweb" class="qrcode"> </div>
+                                </div>
+                              </div>
+                              <div class="font-normal bold text-gray-500 text-center dark:text-white mb-5">{{ w.name }} MWEB Address: </div>
+                              <div class="text-center relative">
+                                <div data-tooltip-target="tooltip-copy-litecoin-mweb" class="input-like-container hover:border-blue-500 bg-gray-50 text-gray-900 appearance-none pr-10 dark:bg-gray-500 dark:text-white border border-gray-300 dark:border-gray-400 dark:text-gray-50 dark:placeholder-gray-400 text-sm rounded-lg outline-none focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 focus:ring-0" id="stealth_address">{{ w.mweb_address }}</div> <span class="absolute inset-y-0 right-0 flex items-center pr-3 cursor-pointer" id="copyIcon"></span>
+                              </div>
+                              <div id="tooltip-copy-litecoin-mweb" role="tooltip" class="inline-block absolute invisible z-10 py-2 px-3 text-sm font-medium text-white bg-blue-500 rounded-lg shadow-sm opacity-0 transition-opacity duration-300 tooltip">
+                                <p>Copy to clipboard</p>
+                                <div class="tooltip-arrow" data-popper-arrow></div>
+                              </div>
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                      {% endif %} 
+                      {# / LTC #}
+                    </div>
+                  </div>
                 </div>
               </div>
             </div>
           </div>
-        </div>
-      </div>
+  </section>
+  {% if w.cid == '1' %}
+  {# PART #}
+  <script>
+    // Particl Stealh
+    var stealthAddress = "{{ w.stealth_address }}";
+    
+    var qrCodeStealh = new QRCode(document.getElementById("qrcode-stealth"), {
```

Comment:
```suggestion
    var qrCodeStealth = new QRCode(document.getElementById("qrcode-stealth"), {
```


perhaps other places "stealh"